docs/contributing/contributing.rst

```diff
@@ -45,14 +45,16 @@ Finding an Issue
 You can familiarize yourself with the Mautic contribution process by reviewing the list of **issues** on the :xref:`Mautic Docs issues` section. These issues have a relatively limited scope. 
 
 After you have selected an issue on the :xref:`Mautic Docs issues` page, follow the below steps:
- 1. Add a comment indicating that you would like to own the ticket. This is to avoid conflicts with others also working on the issue.
- 2. After a Mautic administrator assigns you the issue, you can modify files and track changes on GitHub using command line utility, your web browser, GitHub Desktop, or Gitpod.
- 3. Submit a pull request, and the Mautic administrator will review your changes.
+
+1. Add a comment indicating that you would like to own the ticket. This is to avoid conflicts with others also working on the issue.
+2. After a Mautic administrator assigns you the issue, you can modify files and track changes on GitHub using command line utility, your web browser, GitHub Desktop, or Gitpod.
```

Comment:
Please fix the Vale errors - need to add Gitpod to the accept.txt file as it doesn't recognise it.